package.json

```diff
@@ -4,10 +4,11 @@
   "description": "[F]ind [R]eal [E]states [d]amn eas[y].",
   "scripts": {
     "prepare": "husky",
-    "start": "node prod.js",
-    "dev": "yarn && rm -rf ./ui/public/* && vite",
-    "ui": "rm -rf ./ui/public/* && vite",
-    "prod": "yarn && vite build --emptyOutDir",
+    "start:backend": "NODE_ENV=production node index.js",
+    "start:backend:dev": "nodemon  --watch index.js --watch lib",
```

Comment:
Nodemon uses the `package.json`'s `main` property by default (`index.js` in our case).